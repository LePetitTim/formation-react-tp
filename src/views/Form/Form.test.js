import React from 'react';
import renderer from 'react-test-renderer';
import { Form } from './';

it('should render', () => {
  const tree = renderer.create(<Form match={{ params: { id: '' }}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
