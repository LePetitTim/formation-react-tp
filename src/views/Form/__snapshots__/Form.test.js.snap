// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<form
  className="form"
  onSubmit={[Function]}
>
  <div
    className="form__control"
  >
    <label
      htmlFor="title"
    >
      Title
    </label>
    <input
      id="title"
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div
    className="form__control"
  >
    <label
      htmlFor="content"
    >
      Content
    </label>
    <textarea
      id="content"
      onChange={[Function]}
      value=""
    />
  </div>
  <div
    className="form__control form__control--buttons"
  >
    <div
      className="buttons__errors"
    />
    <div
      className="buttons__buttons"
    >
      <button
        type="submit"
      >
        Ajouter
      </button>
    </div>
  </div>
</form>
`;
